### PR TITLE
Update multi-arch_images_build.yml

### PR DESCRIPTION
**Pull request by gauravpbankar**
Raising a PR for building multi-arch images through Github action CI.

Created Github CI which can have github action and qemu statis user on top of x86 arch and we are using binaries of Power and other archs which are enabling us to create Virtual env for those architectures by this way we are building multi-arch images.

Note: To make it work need to create below two secrets in github repo

- QUAY_USER ---> quay username
- QUAY_PWD ---> quay repo password

### CODE REVIEW

**Comment by sarroutbi** [comment]
> Raising a PR for building multi-arch images through Github action CI.
> 
> Created Github CI which can have github action and qemu statis user on top of x86 arch and we are using binaries of Power and other archs which are enabling us to create Virtual env for those architectures by this way we are building multi-arch images.
> 
> Note: To make it work need to create below two secrets in github repo
> 
> * QUAY_USER ---> quay username
> * QUAY_PWD ---> quay repo password

Hello. Can you please provide some information on why this PR is required?

1. First of all, why are the images being pushed to "sec-eng-special"?
2. Why is this required to be in upstream? It seems something very particular for your group ...
3. How can this be tested? If I am not wrong ... it seems you configured it to run on every push ... is this really required?

File: .github/workflows/multi-arch_images_build.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+name: Multiarch build
+
+on:
+  push:
```
Comment (sarroutbi) [inline .github/workflows/multi-arch_images_build.yml:4]: Please, remove this. Including a Github Action that requires setting QUAY USERNAME / PASSWORD is not a good idea. Keep this Github Action to be used manually only

File: .github/workflows/multi-arch_images_build.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        run: buildah images | grep '${{ env.IMAGE_NAME }}'
+
+      - name: Create and add to manifest
+        run: |
+          buildah manifest create ${{ env.IMAGE_NAME }}
+          buildah manifest add ${{ env.IMAGE_NAME }} ${{ env.IMAGE_NAME }}:linux-amd64
+          buildah manifest add ${{ env.IMAGE_NAME }} ${{ env.IMAGE_NAME }}:linux-ppc64le
+          buildah manifest add ${{ env.IMAGE_NAME }} ${{ env.IMAGE_NAME }}:linux-s390x
+          
+     # Authenticate to container image registry to push the image
```
Comment (sarroutbi) [inline .github/workflows/multi-arch_images_build.yml:68]: Indent this comment appropriately

File: .github/workflows/multi-arch_images_build.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 name: Multiarch build
 
 on:
-  push:
+  workflow_dispatch:
```
Comment (sarroutbi) [inline .github/workflows/multi-arch_images_build.yml:4]: I guess this fits better:

on: [workflow_dispatch]

File: .github/workflows/multi-arch_images_build.yml (hunk 1 above)
Comment (gauravpbankar) [reply .github/workflows/multi-arch_images_build.yml]: For setting  QUAY USERNAME / PASSWORD  this is for pushing multi-arch images on repository where we are keeping those images. So this is must to provide credential to push images on repo.

File: .github/workflows/multi-arch_images_build.yml (hunk 2 above)
Comment (gauravpbankar) [reply .github/workflows/multi-arch_images_build.yml]: Added.

**Pull request merged**